ci/unreal-engine.version

```diff
@@ -1,2 +1,2 @@
 HEAD 4.24-SpatialOSUnrealGDK
-HEAD 4.23-SpatialOSUnrealGDK
+HEAD feature/UNR-2988-schema-generation-test-fix
```

Comment:
Engine PR?

Reply:
Coming soon, this PR wasn't supposed to be "public" yet, I wanted to test the 4.23 changes on CI first

Reply:
PR is up: https://github.com/improbableio/UnrealEngine/pull/471

Will change this file back once the engine PR is merged